packages/editor/src/index.ts

```diff
@@ -70,12 +70,20 @@ import { Paragraph } from "./extensions/paragraph";
 import { ClipboardTextSerializer } from "./extensions/clipboard-text-serializer";
 import { Code } from "@tiptap/extension-code";
 import { DateTime } from "./extensions/date-time";
+import { EditorView } from "prosemirror-view";
 
 const CoreExtensions = Object.entries(TiptapCoreExtensions)
   // we will implement our own customized clipboard serializer
   .filter(([name]) => name !== "ClipboardTextSerializer")
   .map(([, extension]) => extension);
 
+EditorView.prototype.updateState = function updateState(state) {
+  //@ts-ignore
```

Comment:
why are you ignoring TS?

Reply:
This is temporary and related to internal api of ProseMirror for which typings are not available. Without adding this piece of code, the editor crashes during development of every refresh. I will probably add a comment here explaining the reason.